src/assets/styles/components/_footer.scss

```diff
@@ -63,3 +66,10 @@
     }
   }
 }
+
+@media (max-width:568px) {
+  .footer-top-col .opp-badge img {
+    width: 139px;
+    top: 0;
+  }
+}
```

Comment:
we have 2 scss files for each component: responsive and for desktop. Please move media queries to responsive footer.scss
also please keep scss hierachy in all css codes. E.g 
` .footer-top-col {
    .opp-badge{
        img {
            width: 139px;
            top: 0;
              }
         }
}`